reproduce/preprocess.py

```diff
@@ -520,6 +525,8 @@ def process_dataset_lowest_face(args, gaze_labels_only=False, force_create=False
                                 bbox = extract_bboxes(faces)
                             elif args.fd_model == "opencv_dnn":
                                 bbox = detect_face_opencv_dnn(net, frame, args.face_detector_confidence)
+                            if not bbox and args.use_facerec:
```

Comment:
looks like your current logic uses face rec whenever there isn't a face found, but FaceRec class had different logic plans in comments

Reply:
Correct, this logic was supposed to have been removed in that last commit. Amending now.